migrations/versions/201603241826_4878972d7e2f_move_abstract_fields_to_sqlalchemy.py

```diff
@@ -0,0 +1,125 @@
+    op.create_index(op.f('ix_abstracts_accepted_type_id'),
+                    'abstracts', ['accepted_type_id'],
+                    unique=False,
+                    schema='event_abstracts')
+    op.create_index(op.f('ix_abstracts_type_id'),
+                    'abstracts', ['type_id'],
+                    unique=False,
+                    schema='event_abstracts')
+    op.create_index(op.f('ix_abstract_field_values_abstract_id'),
+                    'abstract_field_values',
+                    ['abstract_id'],
+                    unique=False,
+                    schema='event_abstracts')
+    op.create_index(op.f('ix_abstract_field_values_contribution_field_id'),
+                    'abstract_field_values',
+                    ['contribution_field_id'],
+                    unique=False,
+                    schema='event_abstracts')
+    op.create_index(op.f('ix_abstracts_event_id'),
+                    'abstracts', ['event_id'],
+                    unique=False,
+                    schema='event_abstracts')
+    op.create_index(op.f('ix_uq_abstracts_legacy_id_event_id'),
+                    'abstracts',
+                    ['legacy_id', 'event_id'],
+                    unique=True,
+                    schema='event_abstracts')
+
+    # indices for 'events' schema
+    op.create_index('ix_uq_abstract_id',
+                    'contributions',
+                    ['abstract_id'],
+                    postgresql_where=sa.text('NOT is_deleted'),
+                    unique=True,
+                    schema='events')
+    op.create_index(op.f('ix_contributions_abstract_id'),
+                    'contributions',
+                    ['abstract_id'],
+                    unique=False,
+                    schema='events')
+
+
+def downgrade():
+    op.drop_index(op.f('ix_contributions_abstract_id'), table_name='contributions', schema='events')
+    op.drop_index('ix_uq_abstract_id', table_name='contributions', schema='events')
+
+    op.drop_index(op.f('ix_uq_abstracts_legacy_id_event_id'), table_name='abstracts', schema='event_abstracts')
+    op.drop_index(op.f('ix_abstracts_event_id'), table_name='abstracts', schema='event_abstracts')
+    op.drop_index(op.f('ix_abstract_field_values_contribution_field_id'),
+                  table_name='abstract_field_values',
+                  schema='event_abstracts')
+    op.drop_index(op.f('ix_abstract_field_values_abstract_id'),
+                  table_name='abstract_field_values',
+                  schema='event_abstracts')
+    op.drop_index(op.f('ix_abstracts_type_id'), table_name='abstracts', schema='event_abstracts')
+    op.drop_index(op.f('ix_abstracts_accepted_type_id'), table_name='abstracts', schema='event_abstracts')
+
+    op.drop_column('contribution_fields', 'legacy_id', schema='events')
+
+    op.drop_table('abstract_field_values', schema='event_abstracts')
+    op.drop_table('abstracts', schema='event_abstracts')
+
+    op.execute(DropSchema('event_abstracts'))
```

Comment:
`cls.__table__.name` gets it from the underlying `Table` object - but `cls.__tablename__` would work too since we always specify an explicit table name